Project1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
```

Comment:
The `body` element is meant to wrap around **all** the content visible in a web browser (vs the `head` which includes info about the HTML document that will **not** be visible in the browser).

Which is all to say, the opening `body` tag here on line 4 should actually be on line 17 (after the closing `head` tag and before the beginning of the `figure` tag).

Basic HTML structure is:
```html
<!DOCTYPE html>
<html>
<head> <!-- stuff ABOUT the document goes here --> </head>
<body>
  <!-- stuff you want to show in the browser go inside here -->    
</body>
</html>
```